libs/wingcompiler/vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    // Some tests use process.chdir which is not supported in vitest's multi-thread mode
+    threads: false,
```

Comment:
I don't think this config is needed in the wingcompiler project (the tests this is referring to are in the winglang proj)